etc/WMAgentConfig.py

```diff
@@ -50,9 +50,13 @@
 # List of BossAir plugins that this agent will use.
 bossAirPlugins = ["CondorPlugin"]
 
-## Plugin with Condor-Python API ...
+# Plugin with Condor-Python API ...
 bossAirPlugins = ["PyCondorPlugin", "CondorPlugin"]
```

Comment:
We had both plugins in this list because some component was crashing or misbehaving, I don't remember right now.
Ok, I'll remove this line specifying both plugins. Thanks